packages/leemons-cache/src/types/redis/queries.js

```diff
@@ -1,14 +1,26 @@
 const { isEmpty } = require('lodash');
 
 function generateKey(key, pluginName) {
-  return `${pluginName}.${key}`;
+  return `{plugin.${pluginName}}.${key}`;
```

Comment:
I've added the curly braces so redis hashes the keys by plugin, and all the multi operations are performed on the same redis cluster node.